pvlib/tests/conftest.py

```diff
@@ -95,6 +95,19 @@ def assert_frame_equal(left, right, **kwargs):
     not has_bsrn_credentials, reason='requires bsrn credentials')
 
 
+try:
+    # Attempt to load SolarAnywhere API key used for testing
+    # pvlib.iotools.get_solaranywhere
+    solaranywhere_api_key = os.environ["SOLARANYWHERE_API_KEY"]
+    has_solaranywhere_credentials = True
+except KeyError:
+    has_solaranywhere_credentials = False
```

Comment:
I don't understand why codecov is claiming that line 102 isn't covered.  The CI logs show that the tests are running, so I don't see why this line wouldn't be getting hit.  No action needed; just recording my confusion...